lib/components/ContentPanel/ContentPanel.module.scss

```diff
@@ -0,0 +1,134 @@
+@import '../../common/color.controls';
+@import '../../common/constants';
+@import '../../common/mixins';
+
+$content-pane-width: 320px;
+$title-font-size: 20px;
```

Comment:
if this is a new common font size, is it possible to add this to a common place?

Reply:
I don't if it is common. We will work in a separate task with the designer to fix the Typography